README.md

```diff
@@ -30,8 +30,13 @@ This repo contains 3 main folders:
 
 *Group 2 [Americas]: (started 2020-07-30) - Thursdays, 7:30pm EDT/EST*
 
-- 2020-07-30 Chapter 1 - Introduction: [Kevin Kent](https://twitter.com/kevin_m_kent)
+- 2020-07-30 Chapter 1 - Introduction: [Kevin Kent](https://twitter.com/kevin_m_kent) ([slides](https://docs.google.com/presentation/d/1oKhqSY6L6yI3yMwB4JB1eduLqu6UWPZ7Y1v32FK042I/edit?usp=sharing))
```

Comment:
Hiya Kevin :) We've been toying with the readme structure a little so that upcoming meetings are listed in the upper section and completed meetings/content are in the bottom section - this manages the readme flow a bit better for multiple cohorts. 

Interesting thought on the "guiding questions" for the upcoming chapters, let me have a thought about how that might best fit in there :)

Reply:
Hey tan - oh ok, that makes sense. thanks for the heads up. And yeah, I don't want the questions to fill up the schedule but I think they would be helpful somewhere.